### ISSUE TITLE
[Auth] Refresh Tokens expire abnormally and API respond with "invalid request"

### ISSUE THREAD

**Issue by josephpage** [xee-lab/xee-api-docs#82]
Almost every 2 days, one of the refresh_tokens expires (out of a total of ~20), and the API respond with a enigmatic :

```json
{"error":"invalid_request"}
```

The only solution is the get a new refresh_token by manually enter the credentials in the consent screen. I don't have precise stats about this issue, but it is too frequent and serious enough for production environments. Happens since ... mmm... 4 months.

Just occurs this morning with a relatively new refresh_token got last week : 2017-08-30 10:37:40 UTC

If the refresh_token is expired for some reasons, the API should return a appropriate and explicit error message, to allow apps to properly handle the errors and send a renewal request to the account owner.

**Comment by josephpage** [comment]
cc @alexjomin

**Comment by alexjomin** [comment]
Do you have the HTTP code of the response ? It will be easier to point it out in the nginx log. Thank you

**Comment by josephpage** [comment]
Mmm I wait for the error to happen again.
Do refresh tokens have an expiration ?

**Comment by josephpage** [comment]
Mmm I remenber refresh_token are valid for 48 hours (https://github.com/xee-lab/xee-api-docs/issues/24)
but why a `{"error":"invalid_request"}` ? the error should be `Invalid authentication` according to documentation... (https://dev.xee.com/doc/api/api/v3/auth/access_token.md)

**Comment by alexjomin** [comment]
yes @josephpage it's seems like a bug. We will give a look and let you know.

**Comment by josephpage** [comment]
Last times it happened : 
- 9/24/2017 10:53
- 9/21/2017 15:06
- 9/20/2017 22:15

**Comment by josephpage** [comment]
Happened many times last week and today...
